README.md

```diff
@@ -57,24 +57,29 @@ $ brew install s3fs
 Examples
 --------
 
-Enter your S3 identity and credential in a file `/path/to/passwd` and set
+The default location for the s3fs password file can be created:
+
+* using a .passwd-s3fs file in the users home directory (i.e. ~/.passwd-s3fs)
+* using the system-wide /etc/passwd-s3fs file
+
+Enter your S3 identity and credential in a file ` ~/.passwd-s3fs` and set
 owner-only permissions:
 
 ```
-echo MYIDENTITY:MYCREDENTIAL > /path/to/passwd
-chmod 600 /path/to/passwd
+echo MYIDENTITY:MYCREDENTIAL >  ~/.passwd-s3fs
+chmod 600  ~/.passwd-s3fs
 ```
 
 Run s3fs with an existing bucket `mybucket` and directory `/path/to/mountpoint`:
 
 ```
-s3fs mybucket /path/to/mountpoint -o passwd_file=/path/to/passwd
+s3fs mybucket /path/to/mountpoint -o passwd_file= ~/.passwd-s3fs
```

Comment:
Does this actually work when you have a space?  Or do you need `-o passwd_file=$HOME/.passwd-s3fs.`?